### PR TITLE
add user history

### PR DESCRIPTION
**Pull request by tkmct**
Add UserActions history funcionality.
need to add test cases in e2e testing.

### CODE REVIEW

File: packages/plasma-light-client/src/LightClient.ts [hunk 1]
```diff
@@ -759,6 +815,34 @@ export default class LightClient {
     )
   }
 
+  public async getAllUserActions(): Promise<UserAction[]> {
+    const result: UserAction[] = []
+    const currentBlockNumber = await this.commitmentContract.getCurrentBlock()
+    let blockNumber = JSBI.BigInt(0)
+    while (JSBI.greaterThan(blockNumber, currentBlockNumber.data)) {
```
Comment (syuhei176) [inline packages/plasma-light-client/src/LightClient.ts:822]: This loop runs until the condition `(blockNumber > currentBlockNumber)` is true?

File: packages/plasma-light-client/src/UserAction.ts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    const blockNumber = struct.data[3].value as BigNumber
+    return new UserAction(
+      type,
+      Range.fromStruct(range),
+      counterParty,
+      blockNumber
+    )
+  }
+
+  public get amount(): JSBI {
```
Comment (syuhei176) [inline packages/plasma-light-client/src/UserAction.ts:112]: Will we use JSBI for public interface? OK, I'll follow this!

File: packages/plasma-light-client/src/LightClient.ts [hunk 3]
```diff
@@ -759,6 +815,34 @@ export default class LightClient {
     )
   }
 
+  public async getAllUserActions(): Promise<UserAction[]> {
```
Comment (syuhei176) [inline packages/plasma-light-client/src/LightClient.ts:818]: Getting all history seems usable method! Can you write method description for public API?

File: packages/plasma-light-client/src/LightClient.ts (hunk 1 above)
Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts:822]: Yes

File: packages/plasma-light-client/src/UserAction.ts (hunk 2 above)
Comment (tkmct) [reply packages/plasma-light-client/src/UserAction.ts:112]: Let's use JSBI.

File: packages/plasma-light-client/src/LightClient.ts (hunk 1 above)
Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts:822]: ooops, this should be lessThanOrEqual. sorry

**Pull request merged**